examples/rag-spring-article/README.md

```diff
@@ -0,0 +1 @@
+Code for the [Java RAG made easy with SpringAI + Elasticsearch](TODO insert link) article
```

Comment:
TODO: insert link 😆 

And maybe add a line explaining what the article is about.

Reply:
well the link doesn't exist yet, I'm waiting ^^" okay for the description!